librz/core/cmd/cmd_analysis.c

```diff
@@ -7998,3 +7941,35 @@ RZ_IPI RzCmdStatus rz_analyze_value_to_maps_handler(RzCore *core, int argc, cons
 	rz_core_analysis_value_pointers(core, state->mode);
 	return RZ_CMD_STATUS_OK;
 }
+
+RZ_IPI RzCmdStatus rz_analysis_info_show_handler(RzCore *core, int argc, const char **argv, RzCmdStateOutput *state) {
+   cmd_address_info(core, core->offset, state);
```

Comment:
well, with the right indentation :D sorry i did not provide that but it is harder to indent code within github comments.